students/charlie_england/lesson01/assignment/test_unit.py

```diff
@@ -0,0 +1,82 @@
+import sys
+import unittest
+sys.path.insert(0,'./inventory_management')
```

Comment:
Fixing the imports above and running this from assignment will help you not need things like this